src/azure/ApiCenter/ApiCenterRestAPIs.ts

```diff
@@ -37,4 +37,7 @@ export const APICenterDataPlaneRestAPIs = {
     ListApiVersions: (domain: string, apiName: string) => `https://${domain}/workspaces/default/apis/${apiName}/versions`,
     ListApiDefinitions: (domain: string, apiName: string, apiVersion: string) => `https://${domain}/workspaces/default/apis/${apiName}/versions/${apiVersion}/definitions`,
     ExportApiDefinitions: (domain: string, apiName: string, apiVersion: string, definitionName: string) => `https://${domain}/workspaces/default/apis/${apiName}/versions/${apiVersion}/definitions/${definitionName}:exportSpecification`,
+    ListAuthentication: (domain: string, apiName: string, apiVersion: string) => `https://${domain}/workspaces/default/apis/${apiName}/versions/${apiVersion}/securityRequirements`,
+    GetCredential: (domain: string, apiName: string, apiVersion: string, authenticationName: string) => `https://${domain}/workspaces/default/apis/${apiName}/versions/${apiVersion}/securityRequirements/${authenticationName}:getCredentials`,
+    ListApiDeployments: (domain: string, apiName: string) => `https://${domain}/workspaces/default/apis/${apiName}/deployments`,
```

Comment:
this url has no `apiversion`?

Reply:
it's better to add the `apiVersion` since the MSDN take `2024-02-01-preview` as `apiVersion`
https://learn.microsoft.com/en-us/rest/api/dataplane/apicenter/api-deployments/get-deployment?view=rest-dataplane-apicenter-2024-02-01-preview&tabs=HTTP

Reply:
https://learn.microsoft.com/en-us/rest/api/dataplane/apicenter/api-deployments/list-deployments?view=rest-dataplane-apicenter-2024-02-01-preview&tabs=HTTP